docs/fast_data/configuration/projections.md

```diff
@@ -6,36 +6,36 @@ sidebar_label: Projections
 
 In this document we guide you through the configuration of [Projections](/fast_data/the_basics.md#projection) directly in the Console.
 
-## Create a System of Records
+## Create a System of Record
 
-To create a Projection, you should first of all create a System of Records, which is the data source that updates the Projection.
+To create a Projection, you should first of all create a System of Record, which is the data source that updates the Projection.
 
-To do so, open the **Projections** section in the Fast Data group of Mia-Platform Console. Then, select the `Create new System of Record` button.
+To do so, open the **Systems of Record** section in the Fast Data group of Mia-Platform Console. Then, select the `Create new System of Record` button.
 
-The creation of a System of Records requires you to insert a System ID, which is basically the name to recognize the System, and to choose the [Kafka message adapter](/fast_data/configuration/realtime_updater.md#kafka-adapters-kafka-messages-format). 
+The creation of a System of Record requires you to insert a System ID, which is basically the name to recognize the System, and to choose the [Kafka message adapter](/fast_data/configuration/realtime_updater.md#kafka-adapters-kafka-messages-format). 
 
-![Create a System of Records](../img/create_system_of_record.png)
+![Create a System of Record](../img/create_system_of_record.png)
 
-The System of Records is then created.
+The System of Record is then created.
 
 :::note
 In case it is not possible to find the button `Create new System of Record`, it means that a project may have been configured
 to expose Systems of Record under the [Data Catalog](/fast_data/data_catalog/overview.md) feature, which allows to visualize them in a read-only fashion.
 :::
 
-## Delete a System of Records
+## Delete a System of Record
 
-To delete a System of Records, you have to click the `Delete` button in the bottom-right corner of the System of Records detail page.
+To delete a System of Record, you have to click the `Delete` button in the bottom-right corner of the System of Record detail page.
 
 :::warning
 The deletion is not allowed as long as you have at least one Projection inside the System, hence you need to delete all the Projections in a System before being able to delete it. 
 :::
 
 ## Create a Projection
 
-To create a new Projection, open the **Projections** section in the Fast Data group of Mia-Platform Console, then select an existing System of Records or [create a new one](#create-a-system-of-records).
+To create a new Projection, open the **Systems of Record** section in the Fast Data group of Mia-Platform Console, then select an existing System of Record or [create a new one](#create-a-system-of-records).
```

Comment:
```suggestion
To create a new Projection, open the **Systems of Record** section in the Fast Data group of Mia-Platform Console, then select an existing System of Record or [create a new one](#create-a-system-of-record).
```

Since the title has changed, do we need to update also the link?